test/e2e/specs/onboarding/ftme__write.ts

```diff
@@ -41,7 +42,7 @@ describe( DataHelper.createSuiteTitle( 'FTME: Write' ), function () {
 		it( 'Select a .wordpress.com domain name', async function () {
 			const domainSearchComponent = new DomainSearchComponent( page );
 			await domainSearchComponent.search( blogName );
-			await domainSearchComponent.selectDomain( '.wordpress.com' );
+			selectedFreeDomain = await domainSearchComponent.selectDomain( '.wordpress.com' );
```

Comment:
**comment**: I had totally forgotten that I implemented a return value for `domainSearchComponent.selectDomain`. Good use!